packages/trader/src/Modules/Trading/Components/Form/TradeParams/Turbos/turbos-info.jsx

```diff
@@ -0,0 +1,48 @@
+import classNames from 'classnames';
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Money, Text } from '@deriv/components';
+import { Localize } from '@deriv/translations';
+import { connect } from 'Stores/connect';
+
+const getTextForStake = (amount_of_stake, className, currency, text) => (
+    <Text
+        as='p'
+        line_height='s'
+        size='xxxs'
+        className={classNames({
+            [`${className}-tooltip-text`]: className,
+        })}
+    >
+        <Localize
+            i18n_default_text={text}
+            components={[<Money key={0} amount={amount_of_stake} currency={currency} show_currency />]}
+        />
+    </Text>
+);
+
+const TurbosInfo = ({ className, currency, has_stop_loss, max_stake, min_stake }) => {
+    return (
+        <div className={classNames('turbos-trade-info', className)}>
+            {getTextForStake(min_stake, className, currency, 'Min. stake <0/>')}
```

Comment:
```suggestion
            {<TextForStake amount_of_stake={min_stake} className={className} currency={currency} text="Min. stake <0/>" />}
```
I think it looks better as a component